docker-compose.yml

```diff
@@ -8,54 +8,81 @@ services:
     build:
       context: front
       dockerfile: Dockerfile
+      target: prod
+      args:
+        - NODE_ENV=production
+        - API_URL=${API_URL:-http://localhost/api}
+        - WS_BASE_URL=${WS_BASE_URL:-ws://localhost/}
+    expose:
+      - 3000
     env_file:
       - .env
-    ports:
-      - "3001:3000"
-      - "24678:24678"
     depends_on:
       - backend
     restart: always
-    volumes:
-      - ./front/:/app
+    networks:
+      - default
 
   backend:
     container_name: backend
     image: back_image
     build:
       context: back
       dockerfile: Dockerfile
+      target: prod
+      args:
+        - NODE_ENV=production
+    expose:
+      - 3000
     env_file:
       - .env
-    ports:
-      - "3000:3000"
+      - ./back/.env
```

Comment:
```suggestion
```
The Database url being already in the environment from the dotenv, there is no need for the backend one